ev3dev2/motor.py

```diff
@@ -813,10 +813,10 @@ def wait(self, cond, timeout=None):
 
     def wait_until_not_moving(self, timeout=None):
         """
-        Blocks until ``running`` is not in ``self.state`` or ``stalled`` is in
-        ``self.state``.  The condition is checked when there is an I/O event
-        related to the ``state`` attribute.  Exits early when ``timeout``
-        (in milliseconds) is reached.
+        Blocks until ``running`` is not in ``self.state`` or ``stalled``
+        or ``holding` are in ``self.state``.  The condition is checked
```

Comment:
Perhaps a bulleted list would make this more readable?

Reply:
Will do
```
        Blocks until one of the following conditions are met:
        - ``running`` is not in ``self.state``
        - ``stalled`` is in ``self.state``
        - ``holding`` is in ``self.state``
        The condition is checked when there is an I/O event related to
        the ``state`` attribute.  Exits early when ``timeout`` (in
        milliseconds) is reached.
```